align/compiler/find_constraint.py

```diff
@@ -457,7 +417,7 @@ def filter_symblock_const(self, pairs: list):
                 continue
             if not self.G.nodes[key].get("instance"):
                 continue
-            elif "Dcap" in self.G.nodes[key].get("instance").model:
+            elif "DCAP" in self.subckt.get_element(key).model:
```

Comment:
It is not safe to rely on the primitive naming convention. What if a PDK calls it DECAP instead of DCAP? Although this is out of scope of this PR, let's add a "TODO: Do not rely on names" as a reminder for future.

Reply:
By the way, why exclude DCAP's from symmetry? What if there is a DCAP on both differential nets?

Reply:
I do not have an example in our database here but in one of my earlier discussions, we found that DCAPs are generally added in the layout to fill the empty spaces and added to the netlist for LVS purposes. 
This was done before we did not have a do-not-touch constraint for instances. Now when you point it out we can keep it as a user-provided do-not-touch constraint instead of hardcoding in the code.

Reply:
done